pyclibrary/c_parser.py

```diff
@@ -738,7 +437,7 @@ def preprocess(self, path):
         # First join together lines split by \\n
         text = Literal('\\\n').suppress().transformString(text)
 
-        # Define the structure of a macro definition
+        # Macro the structure of a macro definition
```

Comment:
Why were you unhappy with 'Define' ? Perhaps add ':' so that it reads 'Macro : the str...'